Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingsList.swift

```diff
@@ -24,62 +24,94 @@ struct OfferingsList: View {
     fileprivate struct PresentedPaywall: Hashable {
         var offering: Offering
         var mode: PaywallViewMode
+        var responseOfferingID: String
```

Comment:
The `PresentedPaywall` now keeps track of the REST API's ID for this particular offering, so that we can compare it later to new data.